Formula/stellar-core.rb

```diff
@@ -0,0 +1,37 @@
+class StellarCore < Formula
+  desc "The backbone of the Stellar (XLM) network"
+  homepage "https://www.stellar.org/"
+  url "https://github.com/stellar/stellar-core.git",
+    :using => :git,
+    :tag => "v9.1.0",
+    :revision => "a278e95978bdac6d1015d82f4859dad780e752d3"
+  head "https://github.com/stellar/stellar-core.git", :using => :git
+
+  depends_on "pkg-config" => :run
+  depends_on "libsodium" => :run
+  depends_on "libtool" => :run
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "postgresql" => :recommended
```

Comment:
Let's make this mandatory, by removing `=> :recommended`. Fewer options are better for our users, who benefit from prebuilt binaries.

Reply:
Done, thanks.